lib/noble-device.js

```diff
@@ -71,7 +71,9 @@ NobleDevice.prototype.readDataCharacteristic = function(serviceUuid, characteris
 };
 
 NobleDevice.prototype.writeDataCharacteristic = function(serviceUuid, characteristicUuid, data, callback) {
-  this._characteristics[serviceUuid][characteristicUuid].write(data, false, function(error) {
+  var c = this._characteristics[serviceUuid][characteristicUuid];
+  var withoutResponse = c.properties.indexOf('writeWithoutResponse') !== -1;
```

Comment:
Good point! I did not consider the case of both properties being present. Preferring write in this case makes sense. I will make the chang.